frontends/web/src/components/settingsButton/settingsItem.tsx

```diff
@@ -1,26 +1,22 @@
-import { Component } from 'react';
+import { ReactNode } from 'react';
 import style from './settingsButton.module.css';
 
-interface SettingsItemProps {
-    optionalText?: string | null;
-    optionalIcon?: JSX.Element;
+type TProps = {
+  optionalText?: string | null;
+  optionalIcon?: JSX.Element;
+  children: ReactNode;
 }
+export const SettingsItem = ({ optionalText, optionalIcon, children }: TProps) => {
+  return (
+    <div className={[style.container, style.item].join(' ')}>
+      {children}
+      {
+        optionalText && (
+          <span className={style.optionalText}>{optionalText}</span>
+        )
+      }
```

Comment:
since you are here, could you remove the newlines:
`{ optionalText` or  `)}` is more consistent imo and nicer to read.